graderx/views.py

```diff
@@ -153,11 +153,14 @@ def start_grading():
         lab_name = request.args['lab']
     except KeyError:
         return jsonify({'status': "course and lab query parameters must be included"}), 400
-    try:
+    # try:
+    if 'student' in request.args:
+        manager.run_grader(course_name, lab_name, student=True)
+    else:
         manager.run_grader(course_name, lab_name)
-        return "SUCCESS", 200
-    except:
-        return "Failed to run the grader", 500
+    return "SUCCESS", 200
+    # except:
```

Comment:
if we don't need this please remove it

Reply:
Thanks, i actually commented it for debugging and forgot it :joy: